bpf/bpf_overlay.c

```diff
@@ -705,29 +698,6 @@ int cil_from_overlay(struct __ctx_buff *ctx)
 
 	case bpf_htons(ETH_P_IP):
 #ifdef ENABLE_IPV4
-# ifdef ENABLE_HIGH_SCALE_IPCACHE
-#  if defined(ENABLE_DSR) && DSR_ENCAP_MODE == DSR_ENCAP_GENEVE
-		if (ctx_load_meta(ctx, CB_HSIPC_ADDR_V4)) {
```

Comment:
The definitions for `CB_HSIPC_ADDR_V4` and `CB_HSIPC_PORT` can probably go away.